tee-worker/omni-executor/intent/executors/cross-chain/src/lib.rs

```diff
@@ -82,17 +102,21 @@ impl<
 	pub fn new(
 		parentchain_rpc_client_factory: Arc<RpcClientFactory>,
 		transaction_signer: Arc<ParentchainTxSigner>,
-		rpc_endpoint_registry: RpcEndpointRegistry,
+		_rpc_endpoint_registry: RpcEndpointRegistry,
 		pumpx_signer_client: Arc<SignerClient>,
+		pumpx_api: Arc<PumpxApi>,
+		storage_db: Arc<StorageDB>,
 	) -> Result<Self, ()> {
 		// there is no need for account/assets locks if we guarantee the dest-chain payout happens after the source chain finalisation
-		let account_asset_lock = AccountAssetLocks::<AlwaysUnlockedAssetsLock>::empty();
+		// let account_asset_lock = AccountAssetLocks::<AlwaysUnlockedAssetsLock>::empty();
```

Comment:
Is this just to make it compile?
From logic `AlwaysUnlockedAssetsLock` does basically nothing which can be treated as placeholder

Reply:
This specific part no, I commented out everything related to asset lock to be able to compile as we need to refactor it a bit